packages/cffi_example/test_cffi_example.py

```diff
@@ -23,7 +25,11 @@ def test_fnmatch(selenium_module_scope, pattern, name, flags, expected):
     assert result == expected
 
 
-@run_in_pyodide(packages=["cffi_example"], module_scope=True)
+@run_in_pyodide(
+    packages=["cffi_example"],
+    module_scope=True,
+    xfail_browsers={"chrome": "Doesn't work in chrome v90"},
```

Comment:
But it works locally for you with a different chrome version? or it doesn't work in chrome altogether?

Reply:
It works locally on chrome 91 and chrome 96. Breaks locally on Chrome 90.

Reply:
I added a more specific description.